.github/scripts/download_pretrained.py

```diff
@@ -0,0 +1,83 @@
+import argparse
+import time
+from typing import Optional, Text, Tuple
+
+from transformers import AutoTokenizer, TFAutoModel
+
+import rasa.shared.utils.io
+from rasa.nlu.utils.hugging_face.registry import model_weights_defaults
+
+COMP_NAME = "LanguageModelFeaturizer"
+DEFAULT_MODEL_NAME = "bert"
+
+
+def get_model_name_and_weights_from_config(
+    config_path: str,
+) -> Tuple[Optional[Text], Optional[Text]]:
+    config = rasa.shared.utils.io.read_config_file(config_path)
+    print(config)
+    steps = config.get("pipeline", [])
+
+    # Look for LanguageModelFeaturizer
+    steps = list(filter(lambda x: x["name"] == COMP_NAME, steps))
+
+    if len(steps) == 0:
+        print(f"No {COMP_NAME} found")
+        return None, None
+    elif len(steps) > 1:
+        print(f"Too many ({len(steps)}) {COMP_NAME}s found")
+        return None, None
+
+    lmfeat_step = steps[0]
+
+    if "model_name" not in lmfeat_step:
+        return DEFAULT_MODEL_NAME, model_weights_defaults[DEFAULT_MODEL_NAME]
+    model_name = lmfeat_step["model_name"]
+
+    model_weights = lmfeat_step.get("model_weights", model_weights_defaults[model_name])
+
+    return model_name, model_weights
+
+
+def instantiate_to_download(model_weights: Text) -> None:
+    """Instantiates Auto class instances, but only to download."""
+    _ = AutoTokenizer.from_pretrained(model_weights)
```

Comment:
Afaik, this will download to the default cache used by `transformers`. We allow to specify the `cache_dir` in the config though, in which case the model won't be found. To prevent that, the `cache_dir` could be extracted from the config as well and be set in the `from_pretrained` methods, see e.g. [here](https://github.com/huggingface/transformers/blob/v4.16.2/src/transformers/tokenization_utils_base.py#L1555)